ChangeLog.md

```diff
@@ -1,9 +1,21 @@
 # v0.5.0.0
 
+- Derives `Generic1` instances for all non-existentially-quantified effect datatypes.
+
 ## Backwards-incompatible changes
 
 - Replaces `runResource` with an equivalent function that uses `MonadUnliftIO` to select the correct unlifting function (a la `withResource`, which is removed in favor of `runResource`).
 
+- Changes the signature of `eff` from `sig m (m a) -> m a` to `sig m a -> m a`, requiring effects to hold `m k` in their continuation positions instead of merely `k`. This was done in order to improve interoperability with other presentations of higher-order syntax, e.g. `bound`; syntax used with `bound` can now be given `HFunctor` and `Carrier` instances.
```

Comment:
Can we add a short blurb here on how to upgrade? e.g. 

> To upgrade from versions of `fused-effects` < 0.4, convert a handler’s continuations to be `m k` rather than `k`. If no existential type variables appear in your effect definition, you can derive an `HFunctor` and `Effect` instance. Otherwise, implement the required instances by hand.